ops/services/basic_vm/main.tf

```diff
@@ -41,7 +41,7 @@ resource "azurerm_virtual_machine" "vm" {
   }
 
   os_profile_linux_config {
-    disable_password_authentication = false
+    disable_password_authentication = true
```

Comment:
I'm leaving that admin password in place just in case we need some sort of `sudo` password at some point in the workflow once a connection to the box has been established. Disabling the password auth will prevent someone from using a password to log into the box directly, which should minimize the chance of a malicious actor being able to brute force their way in.

Just to confirm no lost functionality, I will make sure I run a bastion test on Dev4 before this gets merged!

Reply:
Confirmed! No lost functionality with the disablement in place. The bastion procedure works just fine.

Reply:
gotcha, thank you!